### PR TITLE
add liondex

### PR DESCRIPTION
**Pull request by LionDEXDev**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
LionDEX

##### Twitter Link:
https://twitter.com/LionDEXOfficial

##### List of audit links if any:
https://github.com/peckshield/publications/blob/master/audit_reports/PeckShield-Audit-Report-LionDEX-v1.0.pdf

##### Website Link:
https://liondex.com/

##### Logo (High resolution, will be shown with rounded borders):
https://drive.google.com/file/d/1fb7Dqus2x-VXy5Ht3M0j2m20mIeVfngv/view?usp=sharing

##### Current TVL:
1.92M

##### Treasury Addresses (if the protocol has treasury)
0x7FcA3BF8AdC4e143BD789AecDa36c0CE34f1d75B 

##### Chain:
arbitrum

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
liondex

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
https://coinmarketcap.com/currencies/liondex/

##### Short Description (to be shown on DefiLlama):
LionDEX is a decentralized crypto derivatives trading protocol. Supported Zero-slippage perpetual futures trading With the innovation of PvP-AMM trading protocol, traders can enjoy extra market liquidity incentives.

##### Token address and ticker if any:
0x8ebb85d53e6955e557b7c53acde1d42fd68561ec    LION

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Derivatives

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
Chainlink and TWAP

##### methodology (what is being counted as tvl, how is tvl being calculated):
Counts USDC in treasure and fsGLP deposited to lionDEXVault. Staking counts Lion deposited to stakingPool.

### CODE REVIEW

File: projects/liondex/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { staking } = require('../helper/staking')
+const { sumTokens2,sumTokensExport } = require('../helper/unwrapLPs')
+
+const treasure = "0x7fca3bf8adc4e143bd789aecda36c0ce34f1d75b";
```
Comment (g1nt0ki) [inline projects/liondex/index.js:4]: hi, this is team multisig? we can count it under treasury metric but not as tvl, sorry.

Comment (LionDEXDev) [reply projects/liondex/index.js:4]: this is the protocol treasury, not for team.   all funds  used for protocol development.

Comment (LionDEXDev) [reply projects/liondex/index.js:4]: hi, is that ok ?  @g1nt0ki

Comment (g1nt0ki) [reply projects/liondex/index.js:4]: hi @LionDEXDev , yes but users can never withdraw this amount right? thats why I think it should be counted as treasury not tvl

Comment (LionDEXDev) [reply projects/liondex/index.js:4]: thanks for your reply, bro. the protocol treasury belong to the community, so the use of funds will be decided through community voting, such as adding liquidity, repurchasing tokens, even withdrawals, etc.   Does this make sense？

**Pull request merged**